src/card_draw.lua

```diff
@@ -0,0 +1,446 @@
+                    self.children.front:draw_shader('debuff', nil, self.ARGS.send_to_shader)
+                end
+            end
+        end
+    end,
+}
+
+SMODS.DrawStep {
+    key = 'greyed',
+    order = 80,
+    func = function(self)
+        if not self.vortex and self.sprite_facing == 'front' then
+            if self.greyed then
+                self.children.center:draw_shader('played', nil, self.ARGS.send_to_shader)
+                if self.children.front and (self.ability.delayed or (self.ability.effect ~= 'Stone Card' and not self.config.center.replace_base_card)) then
+                    self.children.front:draw_shader('played', nil, self.ARGS.send_to_shader)
+                end
+            end
+        end
+    end,
+}
+
+SMODS.DrawStep {
+    key = 'back',
+    order = 0,
+    func = function(self)
+        if not self.vortex and self.sprite_facing == 'back' then
+            local overlay = G.C.WHITE
+            if self.area and self.area.config.type == 'deck' and self.rank > 3 then
+                self.back_overlay = self.back_overlay or {}
+                self.back_overlay[1] = 0.5 + ((#self.area.cards - self.rank)%7)/50
+                self.back_overlay[2] = 0.5 + ((#self.area.cards - self.rank)%7)/50
+                self.back_overlay[3] = 0.5 + ((#self.area.cards - self.rank)%7)/50
+                self.back_overlay[4] = 1
+                overlay = self.back_overlay
+            end
+
+            if self.area and self.area.config.type == 'deck' then
+                self.children.back:draw(overlay)
+            else
+                self.children.back:draw_shader('dissolve')
+            end
+        end
+    end,
+} 
+
+SMODS.DrawStep {
+    key = 'back_sticker',
+    order = 10,
+    func = function(self)
+        if not self.vortex and self.sprite_facing == 'back' then
+            if self.sticker and G.shared_stickers[self.sticker] then
+                G.shared_stickers[self.sticker].role.draw_major = self
+                local sticker_offset = self.sticker_offset or {}
+                G.shared_stickers[self.sticker]:draw_shader('dissolve', nil, nil, true, self.children.center, nil, self.sticker_rotation, sticker_offset.x, sticker_offset.y)
+                if self.sticker == 'Gold' then G.shared_stickers[self.sticker]:draw_shader('voucher', nil, self.ARGS.send_to_shader, true, self.children.center) end
+            end
+        end
+    end,
+}
+
+SMODS.DrawStep {
+    key = 'others',
+    order = 90,
+    func = function(self)
+        for k, v in pairs(self.children) do
+            if not v.custom_draw and k ~= 'focused_ui' and k ~= "front" and k ~= "back" and k ~= "soul_parts" and k ~= "center" and k ~= 'floating_sprite' and k~= "shadow" and k~= "use_button" and k ~= 'buy_button' and k ~= 'buy_and_use_button' and k~= "debuff" and k ~= 'price' and k~= 'particles' and k ~= 'h_popup' then v:draw() end
```

Comment:
This code would probably be cleaner with a table to lookup in and then mods could also add their own values to the table to not be drawn here if they want their own step.